blocks/search/search.css

```diff
@@ -141,9 +141,3 @@ form.search {
         }
     }
 }
-
-
-
-
-
-
```

Comment:
LOL but what if we needed those empty lines.... :-D

Reply:
I'm willing to take my chances they weren't necessary! :D